src/content/syllabus.md

```diff
@@ -151,6 +151,15 @@ response within 24 hours.
   questions or things to discuss with the course staff, please do this
   by emailing TODO. 
 
+### Early Submissions
+Each problem set will have a stated deadline. Additionally, for students who submit more than 24 hours before the deadline, there is a small bonus. The following listed point values are out of a total of 100 points for the assignment.:
+
+At least 48 hours early: +5 point bonus (For a total of +10)
```

Comment:
Having both seems overly complicated to me - have you done it this way before?

Reply:
I think we only had a bonus for at least 48 hours early. Sure, we can keep it simpler and just have a 2 day early bonus.